jenkins/scripts/integration_test.sh

```diff
@@ -47,30 +49,55 @@ if [ "${TESTS_FOR}" == "feature_tests" ]
 then
   TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
 fi
-
 TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-200GB}"
 
-echo "Creating new executer VM."
-
-# Creating new port, needed to immediately get the ip
+# Create new port, needed to immediately get the ip
+echo "Creating a new port to get an IP address."
 EXT_PORT_ID="$(openstack port create -f json \
   --network "${CI_EXT_NET}" \
   --fixed-ip subnet="$(get_subnet_name "${CI_EXT_NET}")" \
   "${TEST_EXECUTER_PORT_NAME}" | jq -r '.id')"
 
-# Create new executer vm
-openstack server create -f json \
-  --image "${IMAGE_NAME}" \
+# Get the base volume
+echo "Getting the base volume."
+BASE_VOLUME_NAME_ID="$(openstack volume show -f json \
+  "${BASE_VOLUME_NAME}" | jq -r '.id')"
+
+# Create test executer volume from copy of base volume
+echo "Creating a test executer volume from copy of base volume."
+TEST_EXECUTER_VOLUME_ID="$(openstack volume create -f json \
+  --source "${BASE_VOLUME_NAME_ID}" \
+  --size 200 \
+  "${TEST_EXECUTER_VM_NAME}" | jq -r '.id')"
+
+if [ "${BASE_VOLUME_NAME}" == "metal3-centos" ]
```

Comment:
@maelk would it be a good idea to try like this?

Reply:
no, this would run on the Jenkins node, not on the test vm. I was talking about the airship-dev-tools volume building script

Reply:
sorry, misunderstood, please check [this](https://github.com/Nordix/airship-dev-tools/pull/217)